docs/site/tutorials/todo-list/todo-list-tutorial-relations.md

```diff
@@ -0,0 +1,141 @@
+---
+lang: en
+title: 'Add Model Relations'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/todo-list-tutorial-relations.html
+summary: LoopBack 4 TodoList Application Tutorial - Add TodoList Repository
+---
+
+### Define the model relation
+
+We are going to add the model relation to indicate the relation that `TodoList`
+_hasMany_ `Todo` using the `lb4 relation` command.
+
+```sh
+$ lb4 relation
+? Please select the relation type hasMany
+? Please select source model TodoList
+? Please select target model Todo
+? Foreign key name to define on the target model todoListId
+? Source property name for the relation getter todos
+? Allow TodoList queries to include data from related Todo instances? Yes
+   create src/controllers/todo-list-todo.controller.ts
+
+Relation HasMany was created in src/
+```
+
+Now, we're going to add the relation for `Todo`. That is, `Todo` _belongsTo_
+`TodoList`.
+
+```sh
+$ lb4 relation
+? Please select the relation type belongsTo
+? Please select source model Todo
+? Please select target model TodoList
+? Source property name for the relation getter todoListId
+? Allow Todo queries to include data from related TodoList instances? Yes
+   create src/controllers/todo-todo-list.controller.ts
+
+Relation BelongsTo was created in src/
+```
+
+### Behind the scene
+
+If you want to understand the code changes introduced from the relation
+generator command, read on the details in this section; otherwise, you are ready
```

Comment:
Suggest:
read on the details in `Relation Generator` https://loopback.io/doc/en/lb4/Relation-generator.html

Reply:
actually what i want to say is if you want to understand more about the code changed introduced from running the command, read this `Behind the scene` section, otherwise, just move on to the next step.  The `Relation Generator` page won't have that details.

Reply:
I see, make sense to me then 👍